src/analyses/variable-sensitivity/full_struct_abstract_object.h

```diff
@@ -32,17 +32,16 @@ class full_struct_abstract_objectt:public struct_abstract_objectt
     const namespacet &ns);
 
 
-  // struct interface
-  virtual abstract_object_pointert read_component(
-    const abstract_environmentt &environment,
-    const member_exprt &member_expr,
-    const namespacet& ns) const override;
+  virtual abstract_object_pointert read(
+    const abstract_environmentt &env,
+    const exprt &specifier,
+    const namespacet &ns) const override;
 
-  virtual sharing_ptrt<struct_abstract_objectt> write_component(
+  virtual abstract_object_pointert write(
```

Comment:
Can you not just not override these and leave the `struct_abstract_object` to forward onto the appropriate virtual methods?